.circleci/config.yml

```diff
@@ -0,0 +1,49 @@
+version: 2.1
+jobs:
+  test-ruby:
+    parameters:
+      ruby_version:
+        type: string
+    docker:
+      - image: "cimg/ruby:<< parameters.ruby_version >>"
+      - image: cimg/redis:6.2.6
+      - image: memcached:1.6.18
+    working_directory: ~/sdk
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: bundle install
+      - run:
+          name: Run tests
+          command: bundle exec rake
+  publish:
+    docker:
+      - image: cimg/ruby:3.2.0
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: bundle install
+      - run:
+          name: Publish to Rubygems
+          command: bundle exec rake release
+workflows:
+  version: 2
```

Comment:
FYI: This version is not required if you are using version 2.1.

Source: https://circleci.com/docs/configuration-reference/#workflow-version

Reply:
What happens when they release 2.2, 3.0, etc?

Reply:
I think the note in the circleci doc is not worded very well. I think the version under workflows is deprecated. But not sure. I guess we can leave it for now.

Reply:
Gil is right, we do not need this version here.

Reply:
I think you removed the wrong version in that last commit. You would want the version at the root level, it is safe to remove the version under `workflows`